backend/src/mirrors_qa_backend/scheduler.py

```diff
@@ -33,33 +35,33 @@ def main():
             for idle_worker in idle_workers:
                 if not idle_worker.countries:
                     logger.info(
-                        f"No countries registered for idle worker {idle_worker.id}"
+                        f"No countries registered for idle worker {idle_worker.id!r}"
```

Comment:
Maybe in the future we should add a custom `__str__` on Worker instead of this this everywhere